metrics/psutils/cpu.lua

```diff
@@ -12,6 +12,11 @@ local instance_file = arg[0]
 
 local threads = {}
 
+local instance_cpu_usage = {
+    system = 0,
+    user = 0
+}
+
 local function update_cpu_metrics()
     collectors_list.cpu_number = utils.set_gauge('cpu_number', 'The number of processors',
         psutils.get_cpu_count(), nil, nil, {default = true})
```

Comment:
It's cumulative, but stime and utime are cumulative as well. So it's the sum of the sum for some reason. I guess it should be

```suggestion
local instance_cpu_usage = {
    system = 0,
    user = 0
}

local function update_cpu_metrics()
    instance_cpu_usage.system = 0
    instance_cpu_usage.user = 0

    collectors_list.cpu_number = utils.set_gauge('cpu_number', 'The number of processors',
        psutils.get_cpu_count(), nil, nil, {default = true})
```

Reply:
Updated.